5.1_task.py

```diff
@@ -0,0 +1,22 @@
+import random
+
+left_operand = []
+right_operand = []
+action_element = []
+
+for i in range(101):
+    left_operand.insert(0, (random.randint(0, 100)))
+    right_operand.insert(0, (random.randint(0, 100)))
+    action_element.insert(0, (random.randint(1, 3)))
```

Comment:
Over-Engineering.

Why don't you just create your_list and append all these variables in one loop?

Also you have list with lenght  101)) because range starts from 0